pnc-core/src/main/java/org/jboss/pnc/core/BuildDriverFactory.java

```diff
@@ -25,7 +25,7 @@ public BuildDriver getBuildDriver(BuildType buildType) throws CoreException {
             }
         }
 
-        throw new CoreException("No build driver available for " + buildType + " build type.");
+        throw new CoreException("No build driver available for " + buildType + " build type."); //TODO create test to make sure exception is properly handled
```

Comment:
I think we should either implement this test straight away (my preference) or if we can't do that for some reason - we should create a JIRA for that - not a TODO in the code. Without JIRA we won't be able to track it.